### ISSUE TITLE
File Browser

### ISSUE THREAD

**Issue by JonathanDCohen** [warpdotdev/Warp#5019]
### Dupe Check

- [X] I have [searched Warp feature requests](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3A%22Feature+requests%22) and there are no duplicates

### Describe the solution you'd like?

A lot of times I find myself using the embedded VSCode terminal just because VSCode has a file browser on the side panel.  Warp is a much better terminal, especially over SSH, and so I'd love to see a file browser.  I don't know if we'd want to manually set where the top level is for the browser, or if it just shows the tree under the pwd at any given time, but it would be a really nice productivity boost.

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Thanks for this feature request! A file browser sounds like a great idea which could boost productivity. 

To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by xtremforce** [comment]
+1 this could be a huge boost

**Comment by j7j7** [comment]
Agreed. remembering filenames / directory structure, can be sometimes hard - having a slide out file browser would be very usful. being able to click on any of them and they pop into the bottom type window. or just launch the in-build code editor.
